configuration/parameters.md

```diff
@@ -960,6 +960,11 @@ Default Value:
 Example:		`Storage-Location=/opt/gravwell/replication`  
 Description:	Sets the storage location for data replicated from other Gravwell indexers.
 
+### **Database-Path**
+Default Value:	`<replication storage location base>/replication.db`
+Example:		`Database-Path=/opt/gravwell/etc/replication.db`  
+Description:	Optional value to store the replication database in an alternate location; by default the replication database will be stored at the base of the location specified in `Storage-Location` as a file named `replication.db` but it can be optionally overridden to an alternate storage location.
```

Comment:
I think we could simplify this as it is a little redundant, but I don't have strong feelings if you want to leave it as is. 

```suggestion
Description:	Optional value to store the replication database in an alternate location. By default the replication database will be stored at the base of the location specified in `Storage-Location` as a file named `replication.db`.
```